rtbkit/core/router/router.cc

```diff
@@ -1417,8 +1451,8 @@ augmentAuction(const std::shared_ptr<AugmentationInfo> & info)
             wakeupMainLoop.signal();
         };
 
-    augmentationLoop.augment(info, Date::now().plusSeconds(augmentationWindow.count()),
-                             onDoneAugmenting);
+    augmentationLoops.at(numAuctions % augmentationLoops.size())->augment(
```

Comment:
Hi @oktal , yes, I'm following a round-robin approach. 
Regarding the at() or [] operator, yes, it's ok, I can add the new modification you are proposing.